.github/workflows/ci.yml

```
@@ -0,0 +1,40 @@
+name: CI
+
+on:
```

I think the conditions here might need adjustment:
```suggestion
on:
  push:
    branches:
      - main
  pull_request:
    branches:
      - main
    types:
      - assigned
      - opened
      - reopened
      - synchronize
```

---

please note - the yaml formatting here might be slightly off, I wrote manually during review

---

Updated although I'm still unsure in what environment the PR action is running and the secrets available

---

Same, I'm tempted to merge this for now (as at least with this setup Github is applying a check to the PR), and continue the research/understanding. Does that work for you?

---

Bit late but yes. I wasn't really happy with the use of pull_request_target and it seems to work ok on the main repo without it

---

Yeah it looks ok, and now its been merged to main the rest of the GH UI (eg actions tab) is functioning correctly.

Thanks again for this contribution, its really appreciated.

No pressure, but if you'd like to collaborate further on this, I've setup a slack workspace for the project which might be an easier way to coordinate any contributions:

https://join.slack.com/t/ngxgovuk/shared_invite/zt-2xmpg6z9h-6q35Y_YD1cLJ5Gznf8hBbg